File: app/src/main/java/ru/SnowVolf/convertx/ui/fragments/base/BaseFragment.java
package ru.SnowVolf.convertx.ui.fragments.base;

import android.graphics.Typeface;
import android.os.Bundle;
import android.view.View;

import androidx.annotation.StringRes;
import androidx.fragment.app.Fragment;

import ru.SnowVolf.convertx.R;

/**
 * Created by Snow Volf on 20.06.2017, 0:43
 */

public class BaseFragment extends Fragment {
    public View rootView;
    public Typeface Mono;
    public BaseFragment(){}
    public @StringRes
    int TITLE = R.string.app_name;


    @Override
    public void onDestroyView(){
        rootView = null;
        super.onDestroyView();
    }

    @Override
    public void onActivityCreated(Bundle savedState){
        super.onActivityCreated(savedState);
        getActivity().setTitle(TITLE);
        Mono = Typeface.createFromAsset(getActivity().getAssets(), "fonts/RobotoMono-Regular.ttf");
    }

    @Override
    public void onResume(){
        super.onResume();
        try {
            getActivity().setTitle(TITLE);
        } catch (Exception ex){
            ex.printStackTrace();
        }
    }
}

File: app/src/main/java/ru/SnowVolf/convertx/ui/fragments/base/BaseFragment.kt
package ru.SnowVolf.convertx.ui.fragments.base

import android.graphics.Typeface
import androidx.annotation.StringRes
import ru.SnowVolf.convertx.R
import android.os.Bundle
import android.view.View
import androidx.fragment.app.Fragment
import java.lang.Exception

/**
 * Created by Snow Volf on 20.06.2017, 0:43
 */
open class BaseFragment : Fragment() {

    lateinit var rootView: View
    @JvmField
    var Mono: Typeface? = null

    @JvmField
    @StringRes
    var TITLE = R.string.app_name
    override fun onDestroyView() {
        super.onDestroyView()
    }

    override fun onActivityCreated(savedState: Bundle?) {
        super.onActivityCreated(savedState)
        requireActivity().setTitle(TITLE)
        Mono = Typeface.createFromAsset(requireActivity().assets, "fonts/RobotoMono-Regular.ttf")
    }

    override fun onResume() {
        super.onResume()
        try {
            requireActivity().setTitle(TITLE)
        } catch (ex: Exception) {
            ex.printStackTrace()
        }
    }
}